website/docs/brain/custom-applications.mdx

```diff
@@ -212,7 +212,43 @@ To visualize the app in your browser, go to http://localhost:8042
 ./uninstall.sh
 ```
 
-### Debugging the App on the Launcher
+## Managing App Ownership and Access
+
```

Comment:
This is more of a hack for when people don't want to use git, right?

Like isn't a better system that each user has their own version of the app on whatever branch they are each developing on?

Reply:
That has been an interesting point of discussion these past few days. I am not sure if it would be better to have 5 different versions of the same custom app in the launcher. IMO, it would be better to have a single app that all users can modify and test. Something more similar to what we do.

Reply:
That's not what we do though. We all have a copy of workspace in our local user account, so we can all develop our own features on our own branches